README.md

```diff
@@ -13,3 +13,18 @@ As a user, I would like to be able to add a feature to my page.
 
 ## What's next?
 Your next task is to update your feature to render server side only. Switch to branch `fusion-05` to see the completed code for the next step.
+
+If you are continuing this training from the last user story, then keep working on your current branch USERNAME-fusion-training.
+
+If you are starting this training from the fifth user story (task Fusion-05), then follow the steps under "What's next?" on the README file in the (https://github.com/wapopartners/Fusion-Training-User-Stories/blob/fusion-01/README.md)[fusion-01] branch - but checkout from fusion-04 instead.
+
+## New Task: User Story 05
+As a user, I would like to render my feature server side only to improve page performance.
+
+## Guide
+1. From root, change directory back into the folder of your feature (from our previous example in fusion-03, that will be `components/features/sample/color-block/`) and open the deafult.jsx file of your component
+
+2. To make a feature run server side, we'll need to set it to static. Using our example, that means adding the following to our component:
+```
+ColorBlock.static = true;
```

Comment:
yep it worked for me, it shows only on the terminal

Reply:
The console log is showing up for me in the browser.

Reply:
Interesting, it writes to the browser console when in the admin but not on the live test. Will approve

Reply:
FUN FACT!

The admin doesn't care about your static files.

Actually, part of the reason I liked this color block approach is because it's really easy to see the two passes (server and then client), and then it also highlights that the admin doesn't care about your precious performance 😂 Cue me telling you a story about how I wrote a popup feature into a static header that needed client interaction and I thought it totally worked when I was testing it in the admin, and then my podmates thought I was crazy when it didn't work.